model/jpa/src/main/resources/META-INF/jpa-changelog-9.0.1.xml

```diff
@@ -45,4 +45,12 @@
         <addUniqueConstraint columnNames="REALM_ID,PARENT_GROUP,NAME" constraintName="SIBLING_NAMES" tableName="KEYCLOAK_GROUP"/>
     </changeSet>
 
+    <changeSet author="keycloak" id="9.0.1-add-index-to-events">
+        <createIndex indexName="IDX_EVENT_TIME" tableName="EVENT_ENTITY">
+            <column name="REALM_ID" type="VARCHAR(255)"/>
+            <column name="EVENT_TIME" type="BIGINT"/>
```

Comment:
@stianst @hmlnarik I am not sure if this is the right place to ask but why didn't we add user id as well? (i.e. `(realm_id, user_id, event_time)`) We are having issues in `auth/realms/{realm}/account/log` page, queries are so slow and sometimes even timed out.